_posts/2023-05-05-welcoming-new-ols-staff.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Welcoming new OLS Staff — Debs and Tajuddeen"
+authors:
+- yochannah
+- npdebs
+---
```

Comment:
For the hero image, maybe something with keywords "welcome" or "welcoming party".
Maybe: https://www.pexels.com/photo/person-standing-near-brown-welcome-on-board-printed-floor-map-128299/

Other location to look for
- Unsplash: https://unsplash.com/s/photos/welcoming-party
- Pexels: https://www.pexels.com/search/welcome/

Few criteria
- Better to have something a bit neutral
- Better to have landscape image

Reply:
This image is perfect! And I know where and how to pick images from next time. Thanks a lot!